### DISCUSSION TITLE
How can I have separate entries for my input fields?

### DISCUSSION THREAD

**Discussion by therealchrismcfly** [neuefische/questions#93] [Web]
Howdy dear Coaches,

**My Problem**

- I added two input fields on every card: 
![Bildschirmfoto 2022-08-22 um 12 09 37](https://user-images.githubusercontent.com/107792731/185896828-c8b1f489-957a-4342-8074-fa548892674d.png)

- My displayed cards are mapped, so the same entries for the input fields appear on every card.
- I would like to have the input fields on every card, but separate entries for each card.

I don't have any clue how I could manage this and I think even my two input fields are working, I wrote one hell of a dirty code. I would be very, very happy to get some help :-)!

Here's the GitHub Link to my current code: https://github.com/therealchrismcfly/capstone-project/tree/addprogress

Thanks a lot in advance! :-)

**Comment by mpagels** [comment]
Ich verstehe leider das Problem nicht. Ich komme mal rum.
